index.html

```diff
@@ -1,84 +1,305 @@
+          </a>
+          <a
+            class="navbar-item-active border-teal-500"
+            href="/scientific-evidence">Scientific Evidence</a>
+          <a
+            class="navbar-item border-yellow-600"
+            href="/practices">Practices</a>
+          <a
+            class="navbar-item border-blue-500"
+            href="/network">Network</a>
+          <a
+            class="navbar-item border-indigo-500"
+            href="/datasets">
+            Datasets
+          </a>
+        </div>
+      </div>
+    </div>
+    <!-- SIDEBAR -->
+    <div id="sidebar"
+      class="absolute left-[117px] top-0 flex h-full w-full max-w-[823px] flex-col bg-white shadow-lg transition-transform duration-500 translate-x-0 z-10">
       <div class="flex grow flex-col overflow-y-auto">
         <div class="space-y-5 p-5 text-slate-700">
           <h2 class="text-3.5xl font-serif">
-            Explore the <span class="font-semibold">impact of different interventions</span> on soil organic carbon
+            Explore the <span class="font-semibold">impact of different
+              interventions</span> on soil organic carbon
           </h2>
         </div>
       </div>
       <div class="absolute -right-10 -ml-px top-0 ">
-        <button class="text-sm font-medium ring-offset-background transition-colors focus-visible:outline-none focus-visible:ring-2 focus-visible:ring-ring focus-visible:ring-offset-2 disabled:pointer-events-none disabled:opacity-50 rounded-none bg-white border-l border-gray-300 justify-center items-center flex h-10 w-10"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="h-6 w-6 transition-transform"><polyline points="15 18 9 12 15 6"></polyline></svg></button>
+        <button id="sidebar-toggle"
+          class="text-sm font-medium ring-offset-background transition-colors focus-visible:outline-none focus-visible:ring-2 focus-visible:ring-ring focus-visible:ring-offset-2 disabled:pointer-events-none disabled:opacity-50 rounded-none border-l border-gray-300 justify-center items-center flex h-10 w-10 btn-theme-dark">
+          <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24"
+            viewBox="0 0 24 24" fill="none" stroke="currentColor"
+            stroke-width="2" stroke-linecap="round" stroke-linejoin="round"
+            class="h-6 w-6 transition-transform"><polyline
+              points="15 18 9 12 15 6"></polyline></svg></button>
       </div>
     </div>
+    <!-- MAP -->
     <div id='map' style='width: 100wh; height: 100vh'></div>
-    <!-- END -->
+    <!-- MAP SETTINGS BUTTON -->
+    <div class="flex flex-col space-y-2 absolute right-6 top-6 z-40">
+      <button
+        id="map-settings-button"
+        class="relative flex h-8 w-8 items-center p-1.5 transition-colors border shadow-sm active:outline active:outline-2 btn-theme-dark"
```

Comment:
The button should be 40x40px and without border nor shadow:
```suggestion
        class="relative flex items-center justify-center w-10 h-10 transition-colors active:outline active:outline-2 btn-theme-dark"
```